packages/commons/test/Deque.AxeCore.Commons.Test.csproj

```diff
@@ -17,6 +17,7 @@
     </PackageReference>
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.0" />
+    <PackageReference Include="Moq" Version="4.18.2" />
```

Comment:
It doesn't look like any of the new tests are using Moq - why the new dependency?

Reply:
Good catch! This was left over from trying out a different unit test which we have since removed. Will remove this as well.